package-lock.json

```diff
@@ -2601,13 +2601,13 @@
     "alphanum-sort": {
       "version": "1.0.2",
       "resolved": "https://registry.npmjs.org/alphanum-sort/-/alphanum-sort-1.0.2.tgz",
-      "integrity": "sha512-0FcBfdcmaumGPQ0qPn7Q5qTgz/ooXgIyp1rf8ik5bGX8mpE2YHjC0P/eyQvxu1GURYQgq9ozf2mteQ5ZD9YiyQ==",
+      "integrity": "sha1-l6ERlkmyEa0zaR2fn0hqjsn74KM=",
```

Comment:
Where these changes come from?

Reply:
![image](https://user-images.githubusercontent.com/1162935/202013404-e4b641e6-3c47-4cba-be4b-85391d573aef.png)

Reply:
![theyreback](https://user-images.githubusercontent.com/1162935/202013694-e2829d4f-a786-4921-b94c-836e6ea3d0b4.gif)

Reply:
@jmiranda Confirmed as an accident after working on the release/0.8.19-hotfix2 branch. No need to panic (yet :grin:).